main.go

```
@@ -98,14 +98,19 @@ func main() {
 	})
 
 	r.HandleFunc("/delivery_form", func(w http.ResponseWriter, r *http.Request) {
-		deliveries, err := deliveryService.All()
+
+		suppliers, err := supplierService.ListAll()
```

Error is ignored here!